src/components/HTMLEngineProvider/HTMLRenderers/MentionReportRenderer/index.tsx

```diff
@@ -56,10 +57,12 @@ function MentionReportRenderer({style, tnode, TDefaultRenderer, reports, ...defa
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const htmlAttributeReportID = tnode.attributes.reportid;
+    const {currentReportID: currentReportIDContext} = useContext(MentionReportContext);
 
     const currentReportID = useCurrentReportID();
+    const currentReportIDValue = currentReportIDContext || currentReportID?.currentReportID;
```

Comment:
The previous PR is reverted because of https://github.com/Expensify/App/issues/48058. The issue is that `currentReportIDContext` is an empty string and previously we used `??`, so `currentReportID?.currentReportID` is never used.

The video showing https://github.com/Expensify/App/issues/48058 is fixed

https://github.com/user-attachments/assets/d62b4a2b-e3ba-4d0b-ae38-15f5409edeea